### PR TITLE
Fix Jupyter Notebook demo instructions

### PR DESCRIPTION
**Pull request by dominiklohmann**
This PR additionally adds a script that runs the demo from a temporary directory.

### CODE REVIEW

File: examples/jupyter/README.md [hunk 1]
```diff
@@ -72,24 +83,24 @@ the Jupyter notebook for local interaction as follows.
 ### Setup
```
Comment (mavam) [inline examples/jupyter/README.md:83]: You could mention that you need to be in `examples/jupyter` for this.

Comment (dominiklohmann) [reply examples/jupyter/README.md:83]: This is the README of `examples/jupyter`.

**Comment by dominiklohmann** [comment]
@mavam please try again after the most recent commit. If that doesn't fix it I'll have to take a look at your machine to see what's messed up with your python setup.

File: examples/jupyter/README.md [hunk 2]
```diff
@@ -5,17 +5,28 @@ This demo provides two Jupyter notebooks to showcase how VAST can be used for
 data analytics, using the python bridge `pyvast`. Follow the instructions below
 to setup a VAST instance with demo data and install dependencies.
 
+## Scripted Demo
+
+Run `./scripts/run-demo` to run the demo in a temporary directory. Note that
+this requires the demo data to be downloaded, which might not be ideal for
+development on the demo, but is practical for a quick demo setup.
+
+Arguments to this script are passed to the 'jupyter notebook' command. Run with
```
Comment (mavam) [inline examples/jupyter/README.md:14]:
```suggestion
Arguments to this script are passed to the `jupyter notebook` command. Run with
```

**Comment by mavam** [comment]
While I didn't get it to work, I think the changes you made are already a big improvement, so feel free to merge.

**Pull request merged**